src/main/java/org/myrobotlab/service/interfaces/AnalogListener.java

```diff
@@ -12,13 +12,13 @@ public interface AnalogListener extends NameProvider {
    * @param name
    *          - sets the name of the access for this AnalogListener e.g. "x"
    */
-  public void setAnalogId(String name);
+  public void setAxis(String name);
```

Comment:
This naming convention seems very joystick specific...

Reply:
not imo ...    these are only generalized naming constructs .. identifiers to an analog axis .. i mean a domain of any analog values can be thought to exist on an axis mathematically .. that's not joystick specific